src/main/java/org/veupathdb/lib/container/jaxrs/model/User.java

```diff
@@ -1,95 +1,31 @@
 package org.veupathdb.lib.container.jaxrs.model;
 
-public class User
-{
-  private long    userID;
-  private String  firstName;
-  private String  middleName;
-  private String  lastName;
-  private String  organization;
-  private String  signature;
-  private String  email;
-  private String  stableID;
-  private boolean isGuest;
+import org.gusdb.oauth2.client.OAuthClient;
+import org.gusdb.oauth2.client.ValidatedToken;
+import org.json.JSONObject;
 
-  public long getUserID() {
-    return userID;
-  }
-
-  public User setUserID(long userID) {
-    this.userID = userID;
-    return this;
-  }
-
-  public String getFirstName() {
-    return firstName;
-  }
-
-  public User setFirstName(String firstName) {
-    this.firstName = firstName;
-    return this;
-  }
-
-  public String getMiddleName() {
-    return middleName;
-  }
-
-  public User setMiddleName(String middleName) {
-    this.middleName = middleName;
-    return this;
-  }
-
-  public String getLastName() {
-    return lastName;
-  }
-
-  public User setLastName(String lastName) {
-    this.lastName = lastName;
-    return this;
-  }
-
-  public String getOrganization() {
-    return organization;
-  }
-
-  public User setOrganization(String organization) {
-    this.organization = organization;
-    return this;
-  }
-
-  public String getSignature() {
-    return signature;
-  }
+/**
+ * Interface for container core users serves two purposes:
+ * 1. Allows services using this library to not directly depend on oauth2 client code
```

Comment:
We still have to take an indirect dependency via container core. Is the idea to avoid directly using classes we don't have a direct dependency on?

Reply:
Oh I guess this also makes the change more backward-compatible